src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicNamingStyleDiagnosticAnalyzer.vb

```diff
@@ -1,11 +1,17 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Collections.Concurrent
 Imports Microsoft.CodeAnalysis.Diagnostics
 Imports Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Diagnostics.Analyzers
     <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
     Friend Class VisualBasicNamingStyleDiagnosticAnalyzer
         Inherits NamingStyleDiagnosticAnalyzerBase
+
+        Protected NotOverridable Overrides Sub OnCompilationStartAction(
+                context As CompilationStartAnalysisContext,
+                idToCachedResult As ConcurrentDictionary(Of Guid, ConcurrentDictionary(Of String, String)))
+        End Sub
```

Comment:
thanks!  *consider* having a comment that says that VB doesn't have to do anythign speical here.  Sometimes a comment is nice though so the next reader realizes that being empty is intentional and not something that was missed.  But totally up to you.

Reply:
fixed in 1afbe495921f65daf8a1cfea43f5416a6a8ff4fa